openerp/openupgrade/deferred_70.py

```diff
@@ -57,8 +57,9 @@ def sync_commercial_fields(cr, pool):
         except Exception:
             good = False
             logger.exception(
-                "Failed syncing commercial fields for partner %d (%s, %s)",
-                partner.id, partner.name, partner.vat,
+                "Failed syncing commercial fields for partner %d %r",
+                partner.id,
+                (partner.name, partner.vat, partner.commercial_partner_id),
```

Comment:
for avoiding more PRs:

```suggestion
                (partner.name, partner.vat, partner.commercial_partner_id, partner.commercial_partner_id.name),
```